shared-libs/cht-script-api/README.md

```diff
@@ -0,0 +1,18 @@
+# CHT Script API
+
+The CHT Script API library is intended to be agnostic and simple. It provides a versioned API from feature modules.
+
+| Function | Arguments | Description |
+| -------- | --------- | ----------- |
+| setChtCoreSettingsDoc | Object, CHT-Core settings DB document | Sets the library's cache to access this document synchronously |
+| setUserSettingsDoc | Object, user settings DB document | Sets the library's cache to access this document synchronously |
+| getApi | | Returns an object of versioned APIs, where every version contains the available functions, for example: `v1.hasPermissions('can_edit')` |
+
+## API v1
+
+The API v1 is defined as follows:
```

Comment:
The idea is to provide more helper functions about different topics (authorization, uhc, etc) and at some point minimise/remove the other globals in tasks, targets and contact-summary. This can be used in `/api`, `/admin` and `/webapp` internally as well. 
I think we're trying out first by providing a way to check for permissions, people can then access them through a `cht` namespace: `const myVar = cht.v1.myFunction();`

(@garethbowen pls correct me as need)

Reply:
is this folder name okay? What do you think? I wanted to differentiate from our `/api` app but naming it `cht-api` is still unclear (I guess 🤔)

What do you think about having the auth code for checking permissions as a section in /cht-script-api ? I mean `/cht-script-api/auth.js` 
it could also be its own thing in `shared-libs/auth` but since it's accessed through this new api, I've put it inside to motivate the access through the new api.

Reply:
I've removed getApi